src/main.ts

```diff
@@ -1,4 +1,5 @@
 import session from "express-session";
+import MongoStore from "connect-mongo";
```

Comment:
I wonder if we need to have both mongoose and connect-mongo?
Can we achieve the same functionality just with mongoose?